plugins/controlboard/src/ControlBoardDriverInteractionMode.cpp

```diff
@@ -40,6 +40,12 @@ bool GazeboYarpControlBoardDriver::getInteractionModes(yarp::dev::InteractionMod
 
 bool GazeboYarpControlBoardDriver::changeInteractionMode(int j, yarp::dev::InteractionModeEnum mode)
 {
+	// The joint is already in the specified interaction mode, return
```

Comment:
Please avoid to mix tabs and spaces, and indent correctly, thanks!
```suggestion
    // The joint is already in the specified interaction mode, return
```

Reply:
I modified. Thanks